medusa/tv/episode.py

```diff
@@ -424,11 +424,16 @@ def status(self):
 
     @status.setter
     def status(self, value):
-        self._status = value
-        self._sync_trakt(value)
+        if value != self._status:
```

Comment:
I don't think this is needed?

Reply:
yeah, it's not.
Well I added it when the trakt call was sstill there, to limit the times it did api calls. But in  the end, it was still spamming traktk calls